lib/components/viewers/RouteRow.js

```diff
@@ -172,42 +164,51 @@ export class RouteRow extends PureComponent {
 
     return (
       <StyledRouteRow ref={this.activeRef}>
-        <RouteDetailsContainer>
-          <OperatorLogo operator={operator} />
-          {route.mode && (
-            <ModeIconElement>
-              <ModeIcon
-                aria-label={getFormattedMode(
-                  getModeFromRoute(route).toLowerCase(),
-                  intl
-                )}
-                height={28}
-                leg={{
-                  routeLongName: route?.longName,
-                  routeShortName: route?.shortName
-                }}
-                mode={getModeFromRoute(route)}
-                role="image"
-                width={28}
-              />
-            </ModeIconElement>
-          )}
-          <RouteName route={route} RouteRenderer={RouteRenderer} />
-        </RouteDetailsContainer>
         <RouteRowButton
-          aria-label={routeMapToggleText}
           className="clear-button-formatting"
           onClick={this._onClick}
           onFocus={this._onFocusOrEnter}
           onMouseEnter={this._onFocusOrEnter}
           onTouchStart={this._onFocusOrEnter}
           patternActive={isActive}
-          title={routeMapToggleText}
-        />
+        >
+          <InvisibleA11yLabel>
+            {isActive &&
+              intl.formatMessage({
+                description: 'Identifies that route is active on map',
+                id: 'components.RouteViewer.selected'
+              })}
+          </InvisibleA11yLabel>
```

Comment:
Should this be an `aria-selected` attribute on the button?

Reply:
Replaced in [865becf](https://github.com/opentripplanner/otp-react-redux/pull/969/commits/865becfc193681477669da87330c8446c3c747f3)! The `aria-pressed` doesn't seem to have a consistent impact on VoiceOver and I'm not sure why, but I think you're right it's more appropriate.